hr_employee_service_length/README.rst

```diff
@@ -0,0 +1,46 @@
+Employee Service Length
+=======================
+
+Calculates the service length of employees
+
+Installation
+============
+
+Install as usual by going to the module install screen
+
+
+Configuration
+=============
+
+No configuration necessary
+
+
+Usage
+=====
+
+
+Known issues / Roadmap
+======================
+
+None
+
+
+Credits
+=======
+
+Contributors
+------------
+* Salton Massally <smassally@idtlabs.sl>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
Can you modify the readme file according the template : https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst